### PR TITLE
Powershell cmdlets for list/get/cancel managed server operation

### PR DESCRIPTION
**Pull request by toki95**
## Description

PowerShell commands for list/get/cancel of managed server management operation.
Also added -AsJob parameter in Set-AzSqlInstance command to be able to test cancel in test. This operation should support execution in the back since it can lock powershell for a couple of hours.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - Azure/azure-powershell-cmdlet-review-pr/issues/510
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by toki95** [comment]
Issue link: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/510
Swagger link: https://github.com/Azure/azure-rest-api-specs/pull/8243

**Comment by toki95** [comment]
Need to publish nuggets first in order for checks to be successfull.

**Comment by msJinLei** [comment]
@toki95 
Sorry for late response.
Please do the following changes
- Rebase to the latest the master as your current change has a conflict with master in Changelog.md. 
- Add the your change description under upcoming release in ./src/Sql/Sql/ChangeLog.md

**Comment by toki95** [comment]
> Rebase to the latest the master as your current change has a conflict with master in Changelog.md.
> Add the your change description under upcoming release in ./src/Sql/Sql/ChangeLog.md

I added entry change description. Please check if everything is ok now.

File: src/Sql/Sql/ManagedInstanceOperation/Cmdlet/CancelAzureSqlManagedInstanceOperation.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+namespace Microsoft.Azure.Commands.Sql.ManagedInstanceOperation.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzSqlInstance cmdlet
+    /// </summary>
+    [Cmdlet("Stop", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceOperation",
+        DefaultParameterSetName = StopByNameAndManagedInstanceAndResourceGroupParameterSet,
+        SupportsShouldProcess = true),
+        OutputType(typeof(AzureSqlManagedInstanceOperationModel))]
+    public class RemoveAzureSqlManagedInstanceOperation : ManagedInstanceOperationCmdletBase
```
Comment (isra-fel) [inline src/Sql/Sql/ManagedInstanceOperation/Cmdlet/CancelAzureSqlManagedInstanceOperation.cs]: The cmdlet is called `stop-` and class name is `Remove` while file name is `Cancel`. We might as well make them all be `Stop` so it won't be confusing

File: src/Sql/Sql/ManagedInstanceOperation/Cmdlet/CancelAzureSqlManagedInstanceOperation.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        /// </summary>
+        [Parameter(
+            ParameterSetName = StopByNameAndManagedInstanceAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 0,
+            HelpMessage = "The name of the operation.")]
+        [Alias("OperationName")]
+        [ResourceNameCompleter("Microsoft.Sql/managedInstance/operations", "ResourceGroupName")]
+        [ValidateNotNullOrEmpty]
+        public System.Guid Name { get; set; }
```
Comment (isra-fel) [inline src/Sql/Sql/ManagedInstanceOperation/Cmdlet/CancelAzureSqlManagedInstanceOperation.cs]: Resource name is always a string, this applies to all cmdlets

File: src/Sql/Sql/help/Set-AzSqlInstance.md [hunk 3]
```diff
@@ -296,21 +311,6 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -MinimalTlsVersion
```
Comment (isra-fel) [inline src/Sql/Sql/help/Set-AzSqlInstance.md]: This shouldn't be deleted, `-MinimalTlsVersion` is still there

**Comment by toki95** [comment]
Abandon PR. After waiting for 20 days after syncing branch I wasn't able to realize what the issue with test is. New PR: https://github.com/Azure/azure-powershell/pull/11452

**Pull request closed without merging**